Bookish/Views/Member/_MemberTable.cshtml

```diff
@@ -0,0 +1,43 @@
+@model List<MemberModel>
+
+<div class="container">
+    <table class="table">
+        <thead>
+        <tr>
+            @foreach (var property in typeof(MemberModel).GetProperties())
+                {
+                    <th scope="col">@property.Name</th>
+                }
+        </tr>
+        </thead>
+        <tbody>
+        @foreach (var member in Model)

```

Comment:
Everything within `<tbody>` should be indented - it's all nested within this block